composer.json

```diff
@@ -6,7 +6,8 @@
     "license": "MIT",
     "require": {
         "goaop/framework": "^1.0|^2.0",
-        "zendframework/zend-modulemanager": "^2.0 | ^3.0"
+        "zendframework/zend-modulemanager": "^2.0 | ^3.0",
+        "symfony/console": "^3.0 || ^4.0"
```

Comment:
What about the idea to put this just in the suggests section because it is not needed to use this module. It is just needed if you want to run the warm up command?

Reply:
Good point. Will change that